module/Education/src/Education/Service/Exam.php

```diff
@@ -408,6 +408,75 @@ public function getTempUploadForm()
         return $this->sm->get('education_form_tempupload');
     }
 
+    /**
+     * Get the add course form.
+     *
+     * @return \Education\Form\AddCourse
+     *
+     * @throws \User\Permissions\NotAllowedException When not allowed to upload
+     */
+    public function getAddCourseForm()
+    {
+        if (!$this->isAllowed('upload')) {
+            $translator = $this->getTranslator();
+            throw new \User\Permissions\NotAllowedException(
+                $translator->translate('You are not allowed to add courses')
+            );
+        }
+        return $this->sm->get('education_form_add_course');
+    }
+
+    /**
+     * Add a new course.
+     *
+     * Will also send an email to the user.
```

Comment:
I do not see email being sent here? You can make a TODO marker for this.

Reply:
This was not supposed to happen. A mistake made copying code from another function.